boards/common/nrf52/Makefile.include

```diff
@@ -24,6 +24,8 @@ ifeq (jlink,$(PROGRAMMER))
     export JLINK_PRE_FLASH := erase\nloadfile $(BINDIR)/softdevice.hex
     export FLASH_ADDR := 0x1f000
     export LINKER_SCRIPT ?= $(RIOTCPU)/$(CPU)/ldscripts/$(CPU_MODEL)_sd.ld
+  else
+    LINKER_SCRIPT ?= cortexm.ld
```

Comment:
I would rather define `LINKER_SCRIPT ?= cortexm.ld` after the `PROGRAMMER` section. It is not really programmer specific.
This way, it would always be defined to this if not set differently for `nordic_softdevice_ble`.

It would also make `nrf52840-mdk` work as currently the programmer is a different one and adding `LINKER_SCRIPT ?= cortexm.ld` in the board feels out of place in the `pyocd` section.

Reply:
But then we should be able to move it to `cpu/nrf52/Makefile.include` without any issues right?

Reply:
It makes even more sense, I was just thinking about this file context in my comment.
But it is more coherent in the same file as the `ROM_LEN` and all definition.

Reply:
Changed!